### ISSUE TITLE
Grouping repositories by organization

### ISSUE THREAD

**Issue by kenfinnigan** [Lyfevest/lyfevest#2]
It would be great if the section where you indicate which repositories you want to accept gigs for was grouped by an organization, and then have the ability to deselect all repositories in an organization.

**Comment by qodfathr** [comment]
Agreed, @kenfinnigan . Easier control over what is on/off is needed. Also, an ability to temporarily turn off everything for a period of time (i.e. "I'll be on vacation so stop bothering me until next Tuesday") may be of value. Thoughts?

**Comment by kenfinnigan** [comment]
I like the temporarily off idea, for holidays or just "taking a break" from everything
